blog/_posts/aria-lanea-blog.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: "Planning for the Future of MABE2"
+date: 2021-08-06
+author: Aria Killibrew Bruehl and Lanea Rohan
+---
+
+## Outline
+
+# Planning for the future of MABE2: A summer of documentation and testing
+​
+Hello! This summer we (Lanea Rohan, '22 Grinnel College, and Aria Killebrew Bruehl,
+'23 Reed College) participated in the WAVES workshop through Michigan State University.
+We dedicated our summer to documenting and testing MABE2, an open-source research platform that provides accessible tools for conducting evolutionary computation and digital evolution research.
+​
+This project was guided by our wonderful mentors Acaica Ackles (documentation) and
```

Comment:
You can link their names to their websites (and even your own names to yours if you like)